1-js/05-data-types/07-map-set-weakmap-weakset/article.md

```diff
@@ -93,22 +93,22 @@ map.set('1', 'str1')
 ```
 ````
 
-## Map from Object
+## Objeden Map
 
-When a `Map` is created, we can pass an array (or another iterable) with key-value pairs, like this:
+Bir `Map` oluşturduğumuzda anahtar-değer çifti olarak array kullanabiliriz:
```

Comment:
array için dizi kelimesini kullanabiliriz. Veri tipleri açısından belki de en iyi bilineni diziler. String'i ben bazen karakter dizisi şeklinde yazıyorum fakat String bu haliyle de belki okunabilir.

Reply:
yeni bir pull req attım